config/locales/hi.yml

```diff
@@ -0,0 +1,41 @@
+
+hi:
+  errors:
+    messages:
+      taken_in_past: पहले लिया गया 
```

Comment:
<img width="1289" alt="Screen Shot 2020-10-08 at 10 50 16 AM" src="https://user-images.githubusercontent.com/1635820/95475284-0e7e6780-0954-11eb-8638-435d86c2293c.png">

This doesn't seem to translate quite right. The English version is `was used previously`.

Reply:
पहले - it can mean both first and past/previous in different contexts. See these - https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=first , https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=previous , https://www.shabdkosh.com/search-dictionary?lc=hi&sl=en&tl=hi&e=past . I will restructure the sentence.

Reply:
Thanks for making a change to this 🧡 

Limited to what Google is telling me, we may be losing a bit of the meaning. Google is showing the updated version, "पहले लिया जा चुका है " to be "Already taken". 

 When I read "already taken" it makes me think more like a username, like someone in the system has already used it. 

However, the goal of this error message is to say you have used this same password in the past and cannot re-use it.

As you mentioned, this is probably Google missing some of the semantics, but I thought it would be worth discussing.